package.json

```diff
@@ -6,12 +6,7 @@
   "author": "Google",
   "license": "Apache-2.0",
   "type": "module",
-  "keywords": [
-    "amp",
-    "stories",
-    "storytelling",
-    "wordpress"
-  ],
+  "keywords": ["amp", "stories", "storytelling", "wordpress"],
```

Comment:
Curious: which version of `node` and `npm` are you using?

Reply:
This runs and passes for me on 
```
$  web-stories-wp git:(task/1998-initial-karma-integration) ✗ node --version
v14.4.0
$  web-stories-wp git:(task/1998-initial-karma-integration) ✗ npm --version
6.14.5
```

Reply:
I'm running the same versions as @carlos-kelly

Reply:
Interesting, thanks for sharing!

I'm only asking because I want to find out why sometimes `keywords` and the like are changed this way.